draft-mouris-cfrg-mastic.md

```diff
@@ -1281,14 +1418,40 @@ security analysis of Mastic is provided in {{MPDST25}}.
 
 # IANA Considerations
 
-TODO
+IANA is requested to add new identifiers to the "Verifiable Distributed
+Aggregation Functions (VDAF)" registry, that is described in {{Section 10 of
+!VDAF}}. In short, VDAF identifiers are used for domain separation, which
+guards against failures of entropy sources, by ensuring that invocations of
+different VDAFs use different derived values, even if they are invoked with
+the same underlying random data.
```

Comment:
I don't think this is needed. (Also, note that we've updated in this text in the latest version: https://cfrg.github.io/draft-irtf-cfrg-vdaf/draft-irtf-cfrg-vdaf.html#section-10-10.)

```suggestion
!VDAF}}.
```